test-binaries/src/bin/client.rs

```diff
@@ -1,19 +1,79 @@
-use ntp_daemon::config::PeerConfig;
-use ntp_proto::SystemConfig;
-use std::error::Error;
+use clap::{Arg, Command};
+use ntp_daemon::ObservableState;
+
+pub const CMD_PEERS: &str = "peers";
+pub const CMD_SYSTEM: &str = "system";
+pub const CMD_CONFIG: &str = "config";
+
+pub const FLAG_LOG_FILTER: &str = "log-filter";
+
+fn build_app<'a>() -> Command<'a> {
+    Command::new("client")
+        // .version(concatcp!(VERSION, "\n"))
+        // .about("Runs the given .roc file, if there are no compilation errors.\nUse one of the SUBCOMMANDS below to do something else!")
```

Comment:
This seems off, also this might be a license issue, please check you own all the code submitted here!